homeassistant/components/fritz/sensor.py

```diff
@@ -90,27 +92,27 @@ def __init__(
         super().__init__(fritzbox_tools, device_friendlyname)
 
     @property
-    def _state_provider(self):
+    def _state_provider(self) -> Any:
```

Comment:
Please try to avoid using `Any` when it's possible.
```suggestion
    def _state_provider(self) -> str | Callable | None:
```